lib/Shift4/Request/SubscriptionUpdateRequest.php

```diff
@@ -68,16 +68,37 @@ public function captureCharges($captureCharges)
         return $this->set('captureCharges', $captureCharges);
     }
 
+    /**
+     * @deprecated For backward compatibility only. Use "getCurrentPeriodEnd()".
+     */
     public function getTrialEnd()
     {
         return $this->get('trialEnd');
     }
 
+    /**
+     * @deprecated For backward compatibility only. Use "currentPeriodEnd($periodEnd)" or "currentPeriodEndNow()".
+     */
     public function trialEnd($trialEnd)
     {
         return $this->set('trialEnd', $trialEnd);
     }
 
+    public function getCurrentPeriodEnd()
+    {
+        return $this->get('currentPeriodEnd');
+    }
+
+    public function currentPeriodEnd($periodEnd)
+    {
+        return $this->set('currentPeriodEnd', $periodEnd);
+    }
+
+    public function currentPeriodEndNow()
```

Comment:
User can stil specify this as currentPeriodEnd('now'), so I believe this method is redundant.
Also it can be misleading without looking into implementation: Do now mean current system time or 'now' keyword?

Reply:
it is just a method that keeps consistency between SDKs, e.g. java sdk also has such method
I believe that by keeping a method that encapsulates our special value inside is still ok